src/search.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+function Search(props) {
+  const [searchInput, setSearchInput] = useState('');
+
+  const handleSearchInput = (event) => {
+    event.preventDefault();
```

Comment:
Since you're passing this function to an `<input onChange>`, you don't need to use `event.preventDefault()` in this case.

Generally speaking, you'll only use it in React to prevent a form from being submitted.